File: api-clients/javascript-closure-angular/API/Client/Pagination.js
goog.provide('API.Client.Pagination');

/**
 * @record
 */
API.Client.Pagination = function() {}

/**
 * Total number of results available.
 * @type {!number}
 * @export
 */
API.Client.Pagination.prototype.total;

/**
 * Number of results to return. Default 10. Max 50.
 * @type {!number}
 * @export
 */
API.Client.Pagination.prototype.count;

/**
 * Number of results per page.
 * @type {!number}
 * @export
 */
API.Client.Pagination.prototype.perPage;

/**
 * Page of results.
 * @type {!number}
 * @export
 */
API.Client.Pagination.prototype.currentPage;

/**
 * Total number of pages of results.
 * @type {!number}
 * @export
 */
API.Client.Pagination.prototype.totalPages;

/**
 * Links to the previous and/or next pages of results.
 * @type {!API.Client.Object}
 * @export
 */
API.Client.Pagination.prototype.links;

File: api-clients/async-scala/src/main/scala/io/swagger/client/model/Pagination.scala
package io.swagger.client.model

import org.joda.time.DateTime
import java.util.UUID


case class Pagination (
  total: Option[Integer],  // Total number of results available.
count: Option[Integer],  // Number of results to return. Default 10. Max 50.
per_page: Option[Integer],  // Number of results per page.
current_page: Option[Integer],  // Page of results.
total_pages: Option[Integer],  // Total number of pages of results.
links: Option[Any]  // Links to the previous and/or next pages of results.
)
